src/js/common/components/questions/Phone.jsx

```diff
@@ -37,6 +37,7 @@ class Phone extends React.Component {
             autocomplete="tel"
             placeholder="xxxxxxxxxx"
             field={this.props.value}
+            required={this.props.required}
```

Comment:
I looked at the other instance of `Phone` and didn't see anything.  And tests are still passing :)